ppr-ui/src/components/common/ContactInformation.vue

```diff
@@ -212,7 +218,7 @@ export default defineComponent({
       required: false
     },
     content: {
-      type: Object as () => ContentIF,
+      type: Object as () => ContactInformationContentIF,
```

Comment:
The point of the generic `ContentIF` was to make a universal interface where you can add/remove any content without any other updates. With this approach, we'd need to update `ContactInformationContentIF` if we want to add/remove content. I'm just not too convinced we need another interface for content, although it makes it _safer_.

Reply:
I felt it made it a bit easier to ensure the all the necessary content is there, specially when a component is used in multiple places. I do like the autocomplete it provides and that it helps avoid spelling errors. I think generally, we can maintain both. With `ContentIF` being a more generic option that when can use initially or for components that are not used in multiple places, and once the Content is further established can create a more specific Content Interface, but I am good either way.

@cameron-eyds any thoughts?

Reply:
I think Dimas point is is very valid and would be the pattern to follow if this component was generic but this component has evolved into something more compound and specific. 
Since `content` prop deconstructs into very specific properties a specific interface seems like it makes sense. 

To add to that, there is a lot of instances where the properties are short circuited before use, having an interface in conjunction with an initialized prop object, you could probably do away with the short circuiting. 

Just my 2 cents, i think both ways work fine!

Reply:
Thanks, let's go ahead with specific IF.